ledger/recovering-indexer-integration-tests/src/test/suite/scala/com/digitalasset/platform/indexer/RecoveringIndexerIntegrationSpec.scala

```diff
@@ -38,6 +38,9 @@ import scala.concurrent.duration.{DurationInt, FiniteDuration}
 import scala.util.Try
 
 class RecoveringIndexerIntegrationSpec extends AsyncWordSpec with Matchers with BeforeAndAfterEach {
+  implicit val asys: ActorSystem = ActorSystem("RecoveringIndexerIntegrationSpec")
+  implicit val mat: Materializer = Materializer(asys)
```

Comment:
I think you can extend `AkkaBeforeAndAfterAll` for this. Not sure about the relative benefits.